### DISCUSSION TITLE
Support MySQL or PostgreSQL?

### DISCUSSION THREAD

**Discussion by hhimanshu** [payloadcms/payload#1941] [Q&A] [answered]
Hello there,

Is there a way to use MySQL/PostgreSQL as a database choice? I am interested in using https://planetscale.com/ as a database.

Thank you

**Comment by cpaczek** [comment]
No

**Comment by zubricks** [comment, marked as answer]
Hey @hhimanshu not at the moment, but that is a item on our roadmap (multiple database support). -> https://github.com/payloadcms/payload/discussions/287
